snap/seed_yaml_test.go

```diff
@@ -81,3 +81,17 @@ func (s *seedYamlTestSuite) TestNoPathAllowed(c *C) {
 	_, err = snap.ReadSeedYaml(fn)
 	c.Assert(err, ErrorMatches, `"foo/bar.snap" must be a filename, not a path`)
 }
+
+var nilSeedItemYaml = []byte(`
```

Comment:
Can you add a test case for the following seed.yaml:
```
snaps:
 -
   name: foo
   file: bar.snap

 -
```
This is close to the format in the ISO and the lone `-` at the end is sufficient to trigger the panic.